lua/lean/init.lua

```diff
@@ -19,7 +20,10 @@ function lean.setup(opts)
   if lsp4.enable ~= false then lean.lsp.enable4(lsp4) end
 
   local infoview = opts.infoview or {}
-  if infoview.enable ~= false then require('lean.infoview').enable(infoview) end
+  if infoview.enable ~= false then
+    if opts.info_pertab then vim.g.lean_info_pertab = true else vim.g.lean_info_pertab = false end
```

Comment:
Maybe just:

```suggestion
    opts.one_per_tab = opts.one_per_tab or true
```

(or the reverse default of `false` if I'm the odd one out here)

And possibly storing the mutable state in the `_infoviews` table is a bit cleaner than using a vim-global option since it's private to this module?